opentok-whiteboard.js

```diff
@@ -310,7 +353,7 @@ var OpenTokWhiteboard = angular.module('opentok-whiteboard', ['opentok'])
                     'signal:otWhiteboard_undo': function (event) {
                         if (event.from.connectionId !== OTSession.session.connection.connectionId) {
                             JSON.parse(event.data).forEach(function (data) {
-                                undoWhiteBoard(data);
+                                undoWhiteBoard(data[1]);
```

Comment:
It doesn't look like the `undoWhiteBoard` method takes the data that is passed in because it overrides the data variable. I don't think undo would be working.